constants/externalAddresses.json

```diff
@@ -36,6 +36,7 @@
       "wbtc": "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
       "usdc": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"
     },
```

Comment:
how do you recommend doing yearn registry address for kovan. mock and deploy?

Reply:
Mocking sounds good